org.lflang/src/org/lflang/util/LFCommand.java

```diff
@@ -42,6 +49,9 @@
  */
 public class LFCommand {
 
+    private static final int PERIOD = 200;
```

Comment:
It would be great to have a brief explanation here as to what exactly these constants control and which unit they are in.